source/reference/color_schemes.rst

```diff
@@ -0,0 +1,256 @@
+=============
+Color Schemes
+=============
+
+
+Overview
+========
+
+Color schemes define the colors
+used to highlight source code in Sublime Text views
+and the colors use for different elements
+found in the editing area:
+background, foreground, selection, caret, etc.
+
+
+File Format
+===========
+
+Color scheme files use the Property List format
+and have the .tmTheme extension.
+The file names of color scheme files
+are displayed in the **Preferences → Color Scheme** menu.
+
+The file format of color schemes
+is inherited from Textmate.
+
+.. note::
+
+   Sublime Text uses the .tmTheme extension for color scheme files
+   to maintain compatibility with Textmate.
+   Confusingly, Sublime Text also has a notion
+   of a user interface (UI) theme.
+   A UI theme is a set of styles and decorations
+   to alter the look of the editor’s UI.
+   It’s important to remember
+   that UI themes and color schemes
+   are two different customization mechanisms.
+   Generally speaking, it is far less complex
+   to create a new color scheme
+   than to create a new UI theme.
+
+
+Where to Store Color Schemes
+============================
+
+You can keep color scheme files anywhere under Packages
+(even inside directories nested multiple levels deep).
+Most frequently, you will find color schemes
+inside their own directory.
+
+By convention, directories containing color scheme files
+have the *Color Scheme -*  prefix.
+For example: *Color Scheme - Default*.
+
+
+Selecting a Color Scheme
+************************
+
+You can change the current color scheme
+by means of the **Preferences → Color Scheme** menu.
+Using settings files or plugins,
+you can create advanced color scheme selection mechanisms
+to change the color scheme
+depending on the file type, time of day, etc.
+
+
+Structure of a Color Scheme File
+================================
+
+Color scheme files are based
+on the Property List format.
+All color scheme files share
+the same top-level structure.
+
+.. insert top-level example here
+
+``name``
+   Optional.
+   Name of the color scheme.
+   Ignored by Sublime Text.
+
+``settings``
+   Array of dict elements.
+   See :ref:`Subelements of Settings` for more information.
+
+``uuid``
+   Optional.
+   A unique identifier for the file. Ignored by Sublime Text.
+
+
+Subelements of Settings
+***********************
+
+Sublime Text supports
+the following color scheme settings:
+
+
+Global Settings
+---------------
+
+Not associated with any scope.
+These settings affect global visual items in the editing area.
+
+``activeGuide``
+   Color of the guide lined up with the caret.
+   Only used if the ``indent_guide_options`` setting
```

Comment:
How I would love to have something like `` :st:set:`indent_guide_options` `` here ;)